src/view.h

```diff
@@ -56,4 +56,6 @@ class Vim;
 - (CGSize)viewSizeFromCellSize:(CGSize)cellSize;
 - (CGSize)cellSizeInsideViewSize:(CGSize)viewSize;
 
+- (BOOL)probablyCommandMode;
```

Comment:
It is necessary because without it redraw.mm does not know about the method. Building it provides a warning and does not insert the preview.  The warning is:
```
src/input.mm:26:35: warning: instance method '-probablyCommandMode' not found
      (return type defaults to 'id') [-Wobjc-method-access]
    else if (mInsertMode || [self probablyCommandMode])
```

If you don't want the `probablyCommandMode` in `view.h`, I can copy the logic over to `redraw.h` (since it is only 1 line). Though if the logic for determining if vim is in command mode ever improves, it will have  to be modified in multiple places.